src/test/java/com/bizo/awsstubs/services/dynamodb2/AttributeValueComparatorTest.java

```diff
@@ -0,0 +1,66 @@
+package com.bizo.awsstubs.services.dynamodb2;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+
+public class AttributeValueComparatorTest {
+
+  private static final String STRING_VALUE_1 = "Alpha";
+  private static final String STRING_VALUE_2 = "Beta";
+  private static final int INT_VALUE_1 = 1;
+  private static final int INT_VALUE_2 = 2;
+
+  @Before
+  public void setUp() {
+  }
+
+  @Test
+  public void test_equal() throws Exception {
+    AttributeValue attribute1 = new AttributeValue().withS(STRING_VALUE_1);
+    AttributeValue attribute2 = new AttributeValue().withS(STRING_VALUE_2);
+
+    boolean result = AttributeValueComparator.equal(attribute1, attribute2);
+    assertFalse(result);
```

Comment:
Ideally each of these tests (for equal, for less than, for greater than) would have both an assertFalse for the negative case + an assertTrue for the positive case.